auth/services/Util.ts

```diff
@@ -0,0 +1,39 @@
+import * as bcrypt from "bcrypt";
+
+// Modified Version of http://blog.grossman.io/how-to-write-async-await-without-try-catch-blocks-in-javascript/
+export function to(promise: Promise<any>) {
+return promise
+  .then((data) => {
+      return [null, data];
+  }).catch((err: Error) =>
+  [err.message, null]
+  );
+}
+
+// Error Web Response
+export async function ReE(res: any, err: any, code: any) {
```

Comment:
I think the more idiomatic way to name these would be onError(), onSuccess()

Reply:
ReE (Response Error) - ReS(Response Success) 
Sure thing I can make it more descriptive!